packages/processor/test/__snapshots__/composition.test.js.snap

```diff
@@ -71,10 +71,31 @@ Object {
 }
 `;
 
-exports[`/processor.js composition should compose from other files 1`] = `
+exports[`/processor.js composition should compose from other files in multiple declarations 1`] = `
+Object {
+  "packages/processor/test/specimens/composes2.css": Object {
+    "composes": Array [
+      "mc_foo",
+      "mc_folder2",
+      "mc_composes",
```

Comment:
This output confuses me. If the input is

```css
.composes {
    composes: folder2 from "./folder/folder2.css";
    composes: foo from "./folder/folder2.css";
}
```

Shouldn't this be `[ "mc_folder2", "mc_foo", "mc_composes" ]`? Why is having `foo` before `folder` correct?

Reply:
That's right, I agree it would be an expected order but for me it works even like this as long as it is consistent because I really don't care about the `mc_composes` position.

I'm not sure how to implement this expected order for multiple `composes` declarations 🤔